<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwoLevelLeague</class>
 <widget class="QWidget" name="TwoLevelLeague">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2-Level League</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>After round:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="roundLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="firstTableTab">
      <attribute name="title">
       <string>First</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="1" column="0" colspan="2">
        <widget class="QTableView" name="firstTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="secondTableTab">
      <attribute name="title">
       <string>Second</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="secondTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="leaguePlayButton">
     <property name="text">
      <string>Play round</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
